src/interfaces/IModularAccount.sol

```diff
@@ -5,7 +5,7 @@ import {ExecutionManifest} from "./IExecutionModule.sol";
 
 type ModuleEntity is bytes24;
 
-type ValidationConfig is bytes26;
+type ValidationConfig is bytes25;
```

Comment:
Nit: would be helpful to describe these UDVTs inline with NatSpec comments.

Reply:
Yep, will port over the comment blocks from the corresponding libraries

Reply:
Another things I find that will be helpful is have a line comments on what the 25 bytes for in the IModularAccount. Each time I have to find the lib and figure out where each flag/data locate.

Reply:
Updated! I think this includes what you're describing @fangting-alchemy